client/components/sidebar-follower-list.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export default class FollowerList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userLoggedIn: JSON.parse(window.localStorage.getItem('account')),
+      followerList: this.props.followerList
```

Comment:
I don't recommend storing this `followerList` in state if you can avoid it. Under most circumstances, it is not a good idea to put a component's props into its own state.